internal/tui/page/chat.go

```diff
@@ -56,6 +57,12 @@ func (p *chatPage) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		if cmd != nil {
 			return p, cmd
 		}
+	case dialog.CommandRunCustomMsg:
+		// Handle custom command execution
+		cmd := p.sendMessage(msg.Content)
```

Comment:
We probably want to check if the agent is busy here before we send the message.
`p.app.CoderAgent.IsBusy()`

Reply:
If it's busy what should the behavior be?

Reply:
I guess just returning a warning should be fine right?

Reply:
I see you already did what I wanted to suggest 😂 awesome.